styles/styles.css

```diff
@@ -0,0 +1,243 @@
+    --grijs: #7A7A7A; /*(voor de service-tegel tekst)*/
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1, h2, h3, h4, h5, h6 {
+    font-family: Merrieweather, serif;
+}
+
+h1 {
+    font-size: 35px;
+}
+
+body {
+    font-family: Roboto, sans-serif;
+}
+
+button, input {
+    font-family: inherit;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-container {
+    flex-basis: 1200px;
+    margin: 0 20px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+
+
+/* --------------- 3.2 Other --------------------------- */
+a,
+a:visited,
+a:link {
+    text-decoration: none;
+    color: var(--lichtgrijs)
+}
+
+button {
+    background-color: var(--roze);
+    border-radius: 30px;
+    border: none;
+    color: var(--lichtgrijs);
+    padding: 18px 35px;
+    margin-top: 15px;
+}
+
+button:hover {
+    cursor: pointer;
+}
+
+/*img {*/
+/*    width: 100%;*/
+/*}*/
+
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+.header-section {
+    background-color: var(--blauw);
+
+}
+
+.header-navigation {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    padding: 24px 0;
+}
+
+.header-navigation ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    flex-basis: 320px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.header-welcome-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: var(--lichtgrijs);
+    width: 600px;
+    margin-top: 100px;
+    text-align: center;
+    gap: 20px;
+
+}
+
+.header-img {
+    display: flex;
+    justify-content: center;
+    padding: 100px 0 0 0;
+    flex-basis: 50px;
+}
+
+/* --------------- 4.3 MAIN ------------------- */
+.main-inner-container {
+    display: flex;
+    flex-direction: column;
+}
+.main-intro-container {
+    display: flex;
+    flex-direction: column;
+    width: 576px;
+
+    color: var(--blauw);
+    font-size: 20px;
+    font-family: 'Merriweather', serif;
+    text-align: center;
+    gap: 20px;
+    margin: 50px 0 50px 0;
+}
+
+/* --------------- 4.3.1 MAIN | ARTICLE SECTIONS------------------- */
+
+.main-intro-container-contact-us {
+    color: var(--zwart);
+    font-size: 16px;
+    font-family: 'Roboto', sans-serif;
+    margin-top: 50px;
+}
+
+.main-intro-container-contact-us a {
+    text-decoration: none;
+    color: var(--blauw)
+}
+
+.main-sections {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
```

Comment:
Het parent element heeft `display: flex;` waardoor je hier `flex: 1;` beter kan gebruiken van width. In mijn dagelijks werk zie ik vaak dat flexbox in combinatie met width/height nooit echt een goeie combi zijn. Als de siblings van dit element ruimte innemen en dit element zelf de overige ruimte mag pakken, dan gaat dat met `width: 100%;` mis, omdat die dan net zo breedt wordt als zijn parent (en er dus geen ruimte overblijft voor de siblings)